sup3r/pipeline/forward_pass.py

```diff
@@ -1585,16 +1647,21 @@ def incremental_check_run(cls, strategy, node_index, chunk_index):
             logger.info('Not running chunk index {}, output file '
                         'exists: {}'.format(chunk_index, out_file))
         else:
-            fwp = cls(strategy, chunk_index, node_index)
-            logger.info(f'Running forward pass for chunk_index={chunk_index}, '
-                        f'node_index={node_index}, '
-                        f'file_paths={fwp.file_paths}')
-            fwp.run_chunk()
+            try:
+                fwp = cls(strategy, chunk_index, node_index)
+                logger.info(f'Running forward pass for '
+                            f'chunk_index={chunk_index}, '
+                            f'node_index={node_index}, '
+                            f'file_paths={fwp.file_paths}')
+                fwp.run_chunk()
+            except Exception as e:
+                msg = ('Sup3r ForwardPass chunk failed!')
+                logger.exception(msg)
+                raise RuntimeError(msg) from e
 
     @classmethod
     def run(cls, strategy, node_index):
         """This routine runs forward passes on all spatiotemporal chunks for
```

Comment:
we lost part of the doc string here think. *for the given node index.

Reply:
oh man oops, either a merge conflict or me fat fingering with vim haha

Reply:
lol no prob